components/registry/org.wso2.carbon.registry.resource.ui/src/main/resources/web/resources/edit_text_content_ajaxprocessor.jsp

```diff
@@ -65,9 +65,9 @@
         <input id="editTextContentIDRichText1" type="radio" name="editTextContentIDRichText" <%=(showPlainText) ? "checked=\"checked\"" : "" %> value="plain" onclick="handleUpdateRichText()" /> <fmt:message key="plain.text.editor"/>
         <input id="editTextContentIDRichText0" type="radio" name="editTextContentIDRichText" <%=(showPlainText) ? "" : "checked=\"checked\"" %> <%=(hideRichText) ? "disabled=\"disabled\"" : "" %> value="rich" onclick="handleUpdateRichText()" /> <fmt:message key="rich.text.editor"/>
     </div>
-    <textarea id="editTextContentIDPlain" style="<%=(showPlainText) ? "" : "display:none;" %>width:99%;height:200px"><%=textContent.replace("&", "&amp;")%></textarea>
+    <textarea id="editTextContentIDPlain" style="<%=(showPlainText) ? "" : "display:none;" %>width:99%;height:200px"><%=Encode.forHtml(textContent.replace("&", "&amp;"))%></textarea>
     <div class="yui-skin-sam" id="editTextContentTextAreaPanel" <%=(showPlainText) ? "style=\"display:none\"" : "" %> >
-        <textarea id="editTextContentID" name="editTextContentID" style="display:none;"><%=textContent.replace("&", "&amp;")%></textarea>
+        <textarea id="editTextContentID" name="editTextContentID" style="display:none;"><%=Encode.forHtml(textContent.replace("&", "&amp;"))%></textarea>
```

Comment:
As Bhathiya had mentioned in the other PR, it's redundant to replace & character because as in [1], forHtml will anyway encode & character.

[1] https://owasp.github.io/owasp-java-encoder/encoder/apidocs/org/owasp/encoder/Encode.html#forHtml-java.lang.String-

Reply:
Thanks Tharindu.. I fixed as suggested.